website/docs/docs/cloud/cloud-cli-installation.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Installing the dbt Cloud CLI (Alpha)
+id: cloud-cli-installation
+description: "Instructions for installing and configuring dbt Cloud CLI"
+---
+
+:::warning Alpha functionality 
+
+The following installation instructions are for the dbt Cloud CLI, currently in alpha. These instructions are not intended for general audiences at this time. 
+
+::: 
+
+## Installing dbt Cloud CLI
+
+### Install and update with Brew (recommended)
+
+Install the CLI: 
+
+```bash
+brew tap dbt-labs/dbt-cli
+brew install dbt-cloud-cli
+```
+
+Verify the installation:
+
+```bash
+which dbt
+dbt --help
+```
+
```

Comment:
Could we make "upgrading" an aside or a callout? It currently feels like a necessary next step.